github/ci/prow-deploy/kustom/base/manifests/local/github-jira-proxy-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt
+  name: github-jira-proxy
+  namespace: default
+spec:
+  tls:
+    - hosts:
+        - prow.ci.kubevirt.io
+      secretName: github-jira-proxy-tls
+  rules:
+    - host: prow.ci.kubevirt.io
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: github-jira-proxy
```

Comment:
Should this be `github-jira-proxy-service` as that is what the service is named below?

Reply:
fixed